_data/projects.json

```diff
@@ -1,7 +1,15 @@
 [{
   "name": "kubernetes-client",
   "description": "Simplified Kubernetes API client for Node.js",
-  "link": "https://www.npmjs.com/package/kubernetes-client",
+  "links": [{
+    "title": "kubernetes-client on GitHub",
+    "type": "source",
+    "url": "https://www.npmjs.com/package/kubernetes-client"
```

Comment:
Update to Github URL : https://github.com/godaddy/kubernetes-client

Reply:
Thanks for catching this!